src/views/Home.vue

```diff
@@ -217,18 +221,21 @@ export default {
               .filter((id) => !(isNullOrWhitespace(id)))
               .map((moduleId) => {
                 const newModule = this.modules.find((module) => module.id === moduleId);
-                if (newModule == null) {
+                if (!newModule) {
                   this.showUnknownModulesError(index + 1, moduleId);
                 }
                 return newModule;
               })
               .filter((module) => module),
           }));
-      }
 
-      const cachedPlan = localStorage.getItem('plan');
-      if (cachedPlan != null) {
-        window.location.hash = cachedPlan;
+        if (newPath !== path) {
+          window.location.hash = newPath.replace('#/', '');
```

Comment:
Is that all that has to be done?

Reply:
Or what is the #/ replacement for?

Reply:
I'm not quite sure what you mean with "Is that all that has to be done?".

Regarding the "/#" I'm not quite sure anymore. I just tested it without the replacement and it worked just fine.
I probably assumed that we had to replace it to not get 2 # in the url. But I'll remove the replacement, since that does not seem to be the case.

Reply:
I misunderstood something, but it's fine now!